### ISSUE TITLE
Generic unofficial event icon

### ISSUE THREAD

**Issue by lgarron** [cubing/icons#103]
https://unofficial.cubing.net/ has a variety of events that don't have icons, and it will have more over time. While I don't mind adding lots of icons (perhaps in a separate "extended" build), there would still always a gap during which an icon is needed but not available yet. A generic icon would be useful for that.

**Comment by dmint789** [comment]
I don't think we really need it. It's more useful to show the event name, since a generic icon could represent anything.
